pixart.js

```diff
@@ -1 +1,30 @@
+var form = document.getElementById('form');
+var input = document.querySelector('#color-field');
+var brush = document.querySelector('.brush');
 
+//another attempt at getting the color to switch
+/*onclick="document.getElementById('brush').style.color = 'red'">*/
+
+//here is a function with the suggested event.preventDefault
+/*$("set-color").click('submit', function(event){
+    event.preventDefault();
+    $(".brush").css("backgroundcolor", "input.value");
+});
+};*/
+
```

Comment:
This actually looks right for the most part, although make sure you pass in the right values into the `.css( )` method. First should be the CSS property (spelled properly) and the second should be the value.